projects/ADuCM360_demo_cn0338/src/Cmd.cpp

```diff
@@ -196,26 +214,12 @@ void CCmdHelp::on_Enter(void)
 void Cmd_ReadData(void)
 {
    while (uart_rx_head != uart_rx_tail) {
+      if (uart_rx_queue[uart_rx_head] >= ' ' && uart_rx_queue[uart_rx_head] <= '~') {
```

Comment:
i think this is longer than 80 chars;
maybe use an intermediate variable for this called `c`; and the line should be fine